linkerd/core/src/main/scala/io/buoyant/linkerd/package.scala

```diff
@@ -39,6 +39,18 @@ import com.twitter.finagle.Stack
  * discovered at runtime with finagle's `LoadService` facility.
  */
 package object linkerd {
+  /**
+   * Reimport [[com.twitter.finagle.buoyant.ParamsMaybeWith ParamsMaybeWith]]
```

Comment:
Instead can we just import `com.twitter.finagle.buoyant.ParamsMaybeWith` where ever it's used?  It shouldn't be THAT many places.